src/Connections/GitlabRepo.ts

```diff
@@ -127,10 +126,10 @@ export interface GitLabTargetFilter {
 }
 
 /**
- * Handles rooms connected to a github repo.
+ * Handles rooms connected to a gitlab repo.
```

Comment:
```suggestion
 * Handles rooms connected to a GitLab repo.
```

Reply:
Now that you mention it, there are a few of these that need updating, so I'll just make a separate PR for them.

Reply:
#428